### ISSUE TITLE
Disable image preview

### ISSUE THREAD

**Issue by LinusGeffarth** [nohana/NohanaImagePicker#72]
Hey, 

Can we somehow disable the user's ability to preview the image? I want the user to only be able to select the image in the collection view.

Thanks for this awesome library!

**Comment by haranicle** [comment]
We cannot disable preview screen now. 😭

**Comment by tylerjames** [comment]
I agree with @LinusGeffarth it would be nice if we could disable the preview and just have tap on the cell to select it rather than to trigger the preview. The checkmark in the corner seems more like a selection indicator rather than a button.

**Comment by ghousesgb** [comment]
Could you kindly add this option not disabling preview screen. corner checkmark is missing in my case. what I am missing.
![screen shot 2017-11-06 at 2 32 29 pm](https://user-images.githubusercontent.com/1427049/32432994-805348e2-c2ff-11e7-8fc4-01818818e366.jpg)

**Comment by haranicle** [comment]
patches welcome. 🌞

**Comment by cagkanciloglu** [comment]
yeah that would be a must feature

**Comment by hell4opt** [comment]
+++
